### PR TITLE
[BACKPORT] [1.11] DCOS-50823 - Mute frequently flakey tests.

### PR DESCRIPTION
**Pull request by jbarrick-mesosphere**
## High-level description

This mutes tests that are known to be flakey.

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS-50823](https://jira.mesosphere.com/browse/DCOS-50823) Mute flakey tests.

## Related tickets (optional)

Other tickets related to this change:

  - [DCOS-46578](https://jira.mesosphere.com/browse/DCOS-46578) DC/OS Enterprise - test_metronome.test_metronome flake - Exception: Job run failed - operation was not completed in 300 seconds.

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change: muting tests
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here: N/A muting tests
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

File: packages/dcos-integration-test/extra/test_metronome.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
+import pytest
+
+
+@pytest.mark.xfailflake(
```
Comment (orsenthil) [inline packages/dcos-integration-test/extra/test_metronome.py]: Only this can be muted. Let's remove the other mutes in this change.

**Comment by jbarrick-mesosphere** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jbarrick-mesosphere** [comment]
@orsenthil changed!

**Comment by orsenthil** [comment]
Please update the description too. Otherwise mergebot updates on those JIRAs might cause alarm.

**Pull request merged**